CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# Contributing to `create-react-app`
+
+♥ `create-react-app` and want to get involved? Thanks! There are plenty of ways you can help!
+
+Please take a moment to review this document in order to make the contribution process easy and effective for everyone involved.
+
+Following these guidelines helps to communicate that you respect the time of the developers managing and developing this open source project. In return, they should reciprocate that respect in addressing your issue or assessing patches and features.
+
+## Core ideas
+
+We do not want any flags or configuration, these will only lead us down a path of becoming the next webpack. We want to find good defaults and actively find ways to improve the developer experience.
```

Comment:
I think the first sentence is a little too harsh to webpack for official doc (I know I said it in comments but still). What we’re trying to say is that adding configuration and flags defeats the purpose of this tool.